build.cmd

```diff
@@ -28,7 +18,7 @@ if not exist .nuget @md .nuget
 @copy %CACHED_NUGET% .nuget\nuget.exe > nul
 
 :Build
-%MSBUILD% build\Build.proj /m /v:m %* /fl /flp:LogFile=msbuild.log;Verbosity=Detailed /nr:false /nologo
+dotnet build src/core/core.csproj --configuration release
```

Comment:
Currently https://github.com/ritterim/automation-storage-emulator/blob/6340f0d5487f36ce809798ad55840b3898404152/build/Build.proj is responsible for building, testing, and packaging, with the latter two currently missing from this PR.

Want to add **running tests** and **creating the NuGet package** as part of `build.cmd`? If not, I can merge this into a branch and finish that up.

Excellent contribution overall. 🎉